### PR TITLE
Use terraform variables instead of hardcoded values

### PR DESCRIPTION
**Pull request by maxf**
Instead of directly put environment-dependent variable values in the terraform
files, we should put them in `tfvars` files, so we know exactly what parameters
differ between environments, and where to find them if they need to be changed
in a different environment.

### CODE REVIEW

**Comment by nacnudus** [comment]
https://trello.com/c/OTRdnMy0/2247-use-terraform-variables-for-oauth

**Comment by maxf** [comment]
I wouldn't use links to trello in commit messages without a human-readable message alongside it.

Many alphagov repos have commit messages only linking to trello boards that don't exist anymore.

**Comment by maxf** [comment]
Making amends and adding description to PR and commit.

**Comment by nacnudus** [comment]
Sorry for the delay reviewing this.

The use of tfvars is good.  I think every variable ought to be defined in every environment, so that only the tfvars files differ between environments, which is the situation that we currently have.

I'm happy to update the PR myself.

File: terraform-staging/govgraphsearch.tf [hunk 1]
```diff
@@ -148,7 +148,7 @@ resource "google_cloud_run_service" "govgraphsearch" {
         }
         env {
           name  = "DISABLE_AUTH"
```
Comment (JazzPote) [inline terraform-staging/govgraphsearch.tf:150]: Optional suggestion. I find positive booleans to be much clearer in the code than negative ones. Especially for feature flags.
Could be for another PR

e.g

```javascript
if (process.env.ENABLE_AUTH) {
    // do
    // auth
    // logic
}```

vs

```javascript
if (!process.env.DISABLE_AUTH) {
    // do
    // auth
    // logic
}```

Comment (maxf) [reply terraform-staging/govgraphsearch.tf:150]: Agreed. It would avoid the double negative here. However things will get a little bit more complicated in the authentication code in the `govuk-knowledge-graph-search` repo. Worth giving it a go though.

**Comment by nacnudus** [comment]
I'm working on an [alternative PR](https://github.com/alphagov/govuk-knowledge-graph-gcp/pull/480) that will address both matters: standardising the config across environments as much as possible, and defining `enable_auth` instead of `disable_auth`.  It'll require another draft PR in https://github.com/alphagov/govuk-knowledge-graph-search.

**Comment by nacnudus** [comment]
Closed by https://github.com/alphagov/govuk-knowledge-graph-gcp/pull/480
